proto/osmosis/tokenfactory/v1beta1/tx.proto

```diff
@@ -63,6 +66,8 @@ message MsgBurn {
     (gogoproto.moretags) = "yaml:\"amount\"",
     (gogoproto.nullable) = false
   ];
+  string burnFromAddress = 3
```

Comment:
this is not related to the send hooks, right? Do we need these mint/burn changes?

Reply:
good catch! Srry meant to address this in a different PR must have been pushed together. Fixed